### ISSUE TITLE
Segfault in groupchat

### ISSUE THREAD

**Issue by fling-** [uTox/uTox#675]
uTox built from https://github.com/uTox/uTox/commit/35db020ba872f45af0c54cd0ff92649b8598d7fd segfaults when groupchat goes online with
```
Toxcore:        Error sending message... 3
Messages:       Unable to update this message...
                msg->disk_offset 0 && m->number 3 receipt_number 2
```

**Comment by fling-** [comment]
Also it crashed my awesome once somehow.

**Comment by GrayHatter** [comment]
> Also it crashed my awesome once somehow.

That's awesome!

Anyone remember when uTox crashed the Linux kernel?

**Comment by GrayHatter** [comment]
I believe this is fixed in #674
